decentraland/components/presentational/__snapshots__/LandForSaleList.test.js.snap

```diff
@@ -1,5 +1,11 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
+exports[`LandForSaleList renders the component after having loaded an empty list 1`] = `
+<LargeText>
+  There are no assets for sale right now.
+</LargeText>
+`;
+
 exports[`LandForSaleList renders the component after loaded an empty list 1`] = `
```

Comment:
@marcelomorgado Hmm, I guess I should delete this manually since this is the one with the old name?

Reply:
I ended up deleting it. Interesting to know that jest doesn't do that automatically

Reply:
I'll do that

Reply:
Oh, I just saw your last message :+1: